.github/workflows/split_monorepo.yml

```diff
@@ -119,7 +119,7 @@ jobs:
       # no tag
       - if: "!startsWith(github.ref, 'refs/tags/')"
         name: Monorepo Split of ${{ matrix.package }}
-        uses: "symplify/monorepo-split-github-action@2.1"
+        uses: danharrin/monorepo-split-github-action@v2.3.0
```

Comment:
Sholud we fork the package into the spiral organization?

Reply:
Why do we need a fork?
TThis is not a fork, the repository has been transferred:
https://github.com/danharrin/monorepo-split-github-action/pull/44#issuecomment-1412209274